sw/device/sca/aes_serial/sca.h

```diff
@@ -0,0 +1,78 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SCA_AES_SERIAL_SCA_H_
+#define OPENTITAN_SW_DEVICE_SCA_AES_SERIAL_SCA_H_
+
+#include "sw/device/lib/dif/dif_uart.h"
+
+#include <stdint.h>
+
+/**
+ * @file
+ * @brief Side-channel analysis support library.
+ */
+
+/**
+ * Initializes the peripherals (pinmux, uart, gpio, and timer) used by SCA code.
+ */
+void sca_init(void);
+
+/**
+ * Returns a handle to the intialized UART device.
+ *
+ * @param[out] uart_out Handle to the initialized UART device.
+ */
+void sca_get_uart(const dif_uart_t **uart_out);
+
+/**
+ * Sets capture trigger high.
+ *
+ * The actual trigger signal used for capture is a combination (logical AND) of:
+ * - GPIO15 enabled here, and
+ * - the busy signal of the AES unit. This signal will go high 40 cycles
+ *   after aes_manual_trigger() is executed below and remain high until
+ *   the operation has finished.
+ */
+void sca_set_trigger_high(void);
+
+/**
+ * Sets capture trigger low.
+ */
+void sca_set_trigger_low(void);
+
+/**
+ * Functions called by `sca_call_and_sleep()` must conform to this prototype.
+ */
+typedef void (*sca_callee)(void);
+
+/**
+ * Calls the given function and puts Ibex to sleep to minimize noise.
+ *
+ * @param callee Function to call before putting Ibex to sleep.
```

Comment:
Also here it would be useful to say something like: "for example trigger operation of the device under test"